Controller/ExceptionController.php

```diff
@@ -126,7 +126,7 @@ protected function getAndCleanOutputBuffering()
      * @param FlattenException $exception
      * @param array            $exceptionMap
      *
-     * @return int|false
+     * @return mixed False, or the $exceptionMap corresponding value
```

Comment:
I replace ``false`` by ``bool`` but the class can send a string, do you know why ?